app/models/tribunal_case.rb

```diff
@@ -28,4 +28,8 @@ def documents(filter: default_documents_filter)
   def default_documents_filter
     [grounds_for_appeal_file_name]
   end
+
+  def company?
```

Comment:
I think this needs a slightly clearer name as well – `TribunalCase#company?` isn't very descriptive of what this actually is. Maybe `TribunalCase#taxpayer_is_company?`?